src/RestWrite.js

```diff
@@ -1104,6 +1102,29 @@ RestWrite.prototype.sanitizedData = function() {
   return Parse._decode(undefined, data);
 }
 
+// Returns an updated copy of the object
+RestWrite.prototype.buildUpdatedObject = function (extraData) {
+  var updatedObject = triggers.inflate(extraData, this.originalData);
```

Comment:
I mean than this does not conform to the guideline because the `var` is visible outside the `if `block:
`function f()
{
  if(test) {
    var v;
  }
}`
... but this is (maybe) OK because var and let have the same scope:

`function f()
{
  var v;
}`

BTW is there a written styling guide for the project ?